### DISCUSSION TITLE
Socket hang up error is thrown for Firefox when using getPuppeteer() in devtools service

### DISCUSSION THREAD

**Discussion by sree-nyt** [webdriverio/webdriverio#9227] [Q&A]
While using devtools getPuppeteer() method throws Socket hang up error while running on Firefox.It runs successfully while running on Chrome.Can you help if we are missing on any setting .

Ps : we are passing moz:debuggerAddress as true for Firefox as specified

**Comment by a-skindzer** [comment]
Get the same error on Chrome browser

wdio v8.10.6

`ErrorEvent {
  [Symbol(kTarget)]: WebSocket {
    _events: [Object: null prototype] { open: [Function], error: [Function] },
    _eventsCount: 2,
    _maxListeners: undefined,
    _binaryType: 'nodebuffer',
    _closeCode: 1006,
    _closeFrameReceived: false,
    _closeFrameSent: false,
    _closeMessage: <Buffer >,
    _closeTimer: null,
    _extensions: {},
    _paused: false,
    _protocol: '',
    _readyState: 3,
    _receiver: null,
    _sender: null,
    _socket: null,
    _bufferedAmount: 0,
    _isServer: false,
    _redirects: 0,
    _url: 'ws://172.17.0.1:4444/session/2439a933c718e0405be421c164bdcdd2/se/cdp',
    _originalIpc: false,
    _originalSecure: false,
    _originalHostOrSocketPath: '172.17.0.1:4444',
    _req: null,
    [Symbol(kCapture)]: false
  },
  [Symbol(kType)]: 'error',
  [Symbol(kError)]: Error: socket hang up
      at connResetException (node:internal/errors:717:14)
      at Socket.socketOnEnd (node:_http_client:526:23)
      at Socket.emit (node:events:525:35)
      at Socket.emit (node:domain:489:12)
      at endReadableNT (node:internal/streams/readable:1359:12)
      at processTicksAndRejections (node:internal/process/task_queues:82:21) {
    code: 'ECONNRESET'
  },
  [Symbol(kMessage)]: 'socket hang up'
}
`
